tests/spec/w3c/headers-spec.js

```diff
@@ -250,19 +250,89 @@ describe("W3C — Headers", function() {
       const doc = await makeRSDoc(ops);
       expect($("#subtitle", doc).length).toEqual(0);
     });
-    
+  
     it("uses existing h2#subtitle as subtitle", async () => {
       const ops = makeStandardOps();
       ops.body = "<h2 id='subtitle'>This is a pre existing subtitle <code>pass</code></h2>" + makeDefaultBody();
+    
+      const doc = await makeRSDoc(ops);
       
+      const subTitleElement = doc.querySelectorAll("h2#subtitle");
+      expect(subTitleElement.length).toEqual(1);
+    
+      const confSubTitle = doc.defaultView.respecConfig.subtitle;
+    
+      if (confSubTitle) {
+        expect(confSubTitle).toEqual("This is a pre existing subtitle pass");
+      } else {
+        expect(confSubTitle).toEqual("");
+      }
+    
+      const [forInnerText] = subTitleElement;
```

Comment:
I'm still not sure what "forInnerText" is? It's clearly an element, but this seems a strange/non-standard way to get at it? Can you help me understand this bit?

Reply:
This `forInnerText` is for checking the entire `innertext` of the subtitle element (normal text + text inside <code\> tags )at once. Now, I think it is becoming a bit repetitive to check this again and again. Shall I remove it ?

Also, I was in college at that time so the late reply :sweat_smile: